src/main/java/io/fabric8/maven/docker/assembly/DockerFileBuilder.java

```diff
@@ -184,6 +192,14 @@ private void addMap(StringBuilder b,DockerFileKeyword keyword, Map<String,String
         }
     }
 
+    private void addArgMap(StringBuilder b,DockerFileKeyword keyword, Map<String,String> map) {
+        for (Map.Entry<String, String> entry : map.entrySet()) {
+            String value = StringUtils.isNotBlank(entry.getValue()) ? "=" + quote(entry.getValue()) : "";
```

Comment:
Currently you can provide them only with a property when running, any variable value defined in `<build>` is used as a *default value*. I think adding it always a default values make sense.

I wonder though whether it might make sense to add an extra configuration section (like `<argValues>` or so) with values to fill in for overwriting these defaults so that different Maven profiles could use different values for this. But thats probably to far fetched, so better stick with the current soluton (just thinking loud ;-). Since this would be an addition it can be always later added later (adding is easy, changing hard)